aspnetcore/publishing/iis.md

```diff
@@ -54,6 +54,9 @@ Proceed through the **Confirmation** step to install the web server role and ser
 
 1. Install the [.NET Core Windows Server Hosting](https://go.microsoft.com/fwlink/?linkid=837808) bundle on the hosting system. The bundle will install the .NET Core Runtime, .NET Core Library, and the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module). The module creates the reverse-proxy between IIS and the Kestrel server. Note: If the system doesn't have an Internet connection, obtain and install the *[Microsoft Visual C++ 2015 Redistributable](https://www.microsoft.com/download/details.aspx?id=53840)* before installing the .NET Core Windows Server Hosting bundle.
 
+    > [!NOTE]
+    > To run ASP.NET templates in Visual Studio 2017, you also need to install the [.NET Core SDK](https://www.microsoft.com/net/download/core).
```

Comment:
The only other ? I have about this is that the SDK should be installed with VS2017 auto-magically, no?